src/decimal.rs

```diff
@@ -52,43 +57,42 @@ const MIN: Decimal = Decimal {
     flags: 2_147_483_648,
     lo: 4_294_967_295,
     mid: 4_294_967_295,
-    hi: 4_294_967_295
+    hi: 4_294_967_295,
 };
 
 #[cfg(feature = "const_fn")]
 const MAX: Decimal = Decimal {
     flags: 0,
     lo: 4_294_967_295,
     mid: 4_294_967_295,
-    hi: 4_294_967_295
+    hi: 4_294_967_295,
 };
 
-
 // Fast access for 10^n where n is 0-9
 static POWERS_10: [u32; 10] = [
-                1,
-               10,
-              100,
-            1_000,
-           10_000,
-          100_000,
-        1_000_000,
-       10_000_000,
-      100_000_000,
+    1,
+    10,
+    100,
+    1_000,
+    10_000,
+    100_000,
+    1_000_000,
```

Comment:
This is one part I disagree with cargo fmt. I'm a big right aligned numbers guy!